app/src/main/java/org/sopt/korailtalk/domain/type/SeatStatusType.kt

```diff
@@ -0,0 +1,18 @@
+package org.sopt.korailtalk.domain.type
+
+enum class SeatStatusType(val isUrgent: Boolean) {
+        AVAILABLE(false),          // 예매가능 - 긴급 아님
+        ALMOST_SOLD_OUT(true),     // 매진임박 - 긴급!
+        SOLD_OUT(false);           // 매진 - 긴급 아님
+
+        companion object {
+                fun from(value: String): SeatStatusType {
+                        return when (value) {
+                                "예매가능" -> AVAILABLE
+                                "매진임박" -> ALMOST_SOLD_OUT
+                                "매진" -> SOLD_OUT
+                                else -> AVAILABLE  // 기본값
+                        }
+                }
+        }
+}
```

Comment:
이것도
```suggestion
enum class SeatStatusType(
val text: String,
val isUrgent: Boolean,
) {
        AVAILABLE("예매가능", false),          // 예매가능 - 긴급 아님
        ALMOST_SOLD_OUT("매진임박", true),     // 매진임박 - 긴급!
        SOLD_OUT("매진", false);           // 매진 - 긴급 아님
```
식으로 하면 더 명확해질 것 같아요.

왜냐면 enum에 올 수 있는 값은 3개가 끝인데, `from` 함수에서 else를 굳이 처리해줘야하나?? 싶긴 해요

Reply:
텍스트로 값 찾는 건 `entries.find()` 활용하고, 앞서 적었던 text로 필터링 조건 걸면 아마 매칭도 쉽게 될 거예요

Reply:
그리고 서버에서도 문자열 아마 enum으로 저장하는 거 같아서 "예매가능", "매진임박" 식의 텍스트로는 안 올거예요!
서버 응답 보면서 매칭 로직을 수정해바도 좋을 것 같습니다ㅎㅎ

그리고 아키텍처적인 측면에서는 저희가 '유사 클린' 구조로 가기로 했으니까 어쨌든 도메인이 순수해야하는데, 서버 문자열 매핑을 도메인 레이어에서 진행하는 게 맞을까?도 생각해보시면 좋을 것 같아요